massa-models/src/config/constants.rs

```diff
@@ -389,6 +389,26 @@ pub const MAX_DENUNCIATIONS_PER_BLOCK_HEADER: u32 = 128;
 pub const ROLL_COUNT_TO_SLASH_ON_DENUNCIATION: u64 = 1;
 /// Maximum size of executed denunciations
 pub const MAX_DENUNCIATION_CHANGES_LENGTH: u64 = 1_000;
+/// Maximum size of deferred call pool changes
+// TODO define this value
+// TODO: set to a reasonable value max pool changes
+pub const DEFERRED_CALL_MAX_POOL_CHANGES: u64 = 100_000;
+/// Maximum size of deferred call future slots (1 week)
+pub const DEFERRED_CALL_MAX_FUTURE_SLOTS: u64 = 1209600;
+/// maximum gas for deferred call
+pub const DEFERRED_CALL_MAX_ASYNC_GAS: u64 = MAX_ASYNC_GAS;
+/// max change denominator  
+pub const DEFERRED_CALL_BASE_FEE_MAX_CHANGE_DENOMINATOR: usize = 8;
+/// deferred call min gas increment (1 nanomassa)
+pub const DEFERRED_CALL_MIN_GAS_INCREMENT: u64 = 1;
+/// deferred call max gas cost (10 nanomassa)
+pub const DEFERRED_CALL_MIN_GAS_COST: u64 = 10;
```

Comment:
Is this DEFERRED_CALL_MIN_GAS_COST or the MAX as written in the commented part?

Reply:
> Is this DEFERRED_CALL_MIN_GAS_COST or the MAX as written in the commented part?

Hello @plouis01, indeed it is an error in the comment, I will update it